### DISCUSSION TITLE
Feature suggestion: function that adding all nodes in a workflow to current workflow

### DISCUSSION THREAD

**Discussion by yikchau** [Comfy-Org/ComfyUI-Manager#641] [Ideas]
so that we can save multiple modular workflow and use them when need.

**Comment by ltdrdata** [comment]
Your suggestion is unclear. Could you please elaborate more?

**Reply by yikchau** [reply]
It is simply importing all nodes in a workflow into the current workflow.
Let's say I am currently working on a faceswap workflow and finaly make it done but I want to upscale after faceswap.
And I previously had finished a perfect upscale workflow.
With this function, I can import all nodes(samplers, controlnets, loras etc.) from the upscale workflow.
Then with few simple links I can connect the faceswap workflow to my upscale workflow.

**Reply by ltdrdata** [reply]
What you're talking about sounds like a template feature.
